rails-commenting.rb

```diff
@@ -10,21 +10,25 @@
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    #  this @posts variable is containing all the information on the BlogPost Table
```

Comment:
@posts is a busy instance variable. Correct for the index method it is containing all the information from the database.